### DISCUSSION TITLE
Lack of defined conceptual model & documentation

### DISCUSSION THREAD

**Discussion by MayteTDGeograma** [gopeg/issues-barriers#46] [General]
When creating cross-border data sets, there are several aspects of the lineage of the input data sets to keep in mind, each of which can limit re-useability.

- **formalised** characteristics (in a formal logical and/or conceptual model): Often, there is no complete definition of the data model and the used code lists available, and/or it is not linked from data or metadata. Instead, this knowledge lies only with the creators and operators of the data set.
- **defined** characteristics (in legal texts or guidelines or quality handbooks): Often, the rules according to which this data set has been created and the implicit impact these have on how objects are formed are not known or referenced from the data or metadata.
- **interpreted** characteristics (how the defined characteristics have been applied):  In many cases, systematic biases are introduced into data by how operators interpreted the rules, or also by limitations of tools that were used. These are rarely documented and analysed at all, and are also not linked from the data or metadata.

Thus, to facilitate re-use and to assess suitability of an (original) data set for re-use, it is needed to establish a standard for the publication of such characteristics. The conceptual model should include domain constraints, e.g. in OCL. For INSPIRE, this formalised level is indicated through the compliance statement.

Documenting these aspects and linking such documentation from the dataset and metadata will significantly increase the understanding and usability of the information provided.

**Comment by AndersOstmanNovogit** [comment]
This issue is currently not addressed in D4.3.

The reason for this is that it has several similarities and overlaps with issue #37 and #30.

Can the issues described here be considered to be covered in #37 and/or #30? If not, shall we increase anyone of these other two or shall we keep this as a separate issue. If we keep it, please describe the problem we have encountered more clearly.
